index.js

```diff
@@ -12,10 +12,10 @@ var regexSequences = [
     // Remove XML stuffs and comments
     [/<\?xml[\s\S]*?>/gi, ""],
     [/<!doctype[\s\S]*?>/gi, ""],
-    [/<!--.*-->/gi, ""],
+    [/<!--[\s\S]*?-->/g, ""],
 
     // SVG XML -> HTML5
-    [/\<([A-Za-z]+)([^\>]*)\/\>/g, "<$1$2></$1>"], // convert self-closing XML SVG nodes to explicitly closed HTML5 SVG nodes
+    [/\<([a-z]+)([^\>]*)\/\>/gi, "<$1$2></$1>"], // convert self-closing XML SVG nodes to explicitly closed HTML5 SVG nodes
```

Comment:
In case anyone else is a little rusty on their RegExps, this is a readability improvement: drop `A-Z` and make `a-z` case-insensitive with the `i` flag.